routemaster/models/subscriber.rb

```diff
@@ -79,6 +79,28 @@ def uuid
       _attributes['uuid']
     end
 
+    def health_points
+      _attributes.fetch('health_points', '100').to_i
+    end
+
+    def change_health_by(offset)
+      new_value = _redis_lua_run(
+        'subscriber_change_health_by',
```

Comment:
This is how all the other scripts are being executed.
Caching the lua scripts on Redis sounds like a good idea, but maybe it's for another PR.